sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/molecules/SelectRowViewScreen.kt

```diff
@@ -0,0 +1,107 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import uk.gov.hmrc.components.compose.molecule.selectrow.SelectRowView
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+import uk.gov.hmrc.sample_compose_components.R
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ExamplesSlot
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.HmrcCard
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.PlaceholderSlot
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ScreenScrollViewColumn
+import uk.gov.hmrc.sample_compose_fragments.presentation.viewModel.SelectRowViewModel
+import uk.gov.hmrc.sample_compose_fragments.util.ComposeUtil.collectAsStateLifecycleAware
+
+object SelectRowViewScreen {
+
+    private const val THIRD_ROW_VIEW = 3
+
+    @Composable
+    operator fun invoke(viewModel: SelectRowViewModel) {
+        val context = LocalContext.current
+
+        val exampleUiState by viewModel.exampleUiState.collectAsStateLifecycleAware()
+        val placeHolderUiState by viewModel.placeHolderUiState.collectAsStateLifecycleAware()
+
+        ScreenScrollViewColumn {
+            //region Place Holder
+            PlaceholderSlot {
+                viewModel.setPlaceHolderErrorText(stringResource(id = R.string.select_row_error_message))
+                SelectRowView(
+                    selectRowViewItems = listOf(stringResource(R.string.select_row_body_description)),
+                    errorText = placeHolderUiState.errorText,
+                    rowSelectedPosition = placeHolderUiState.rowSelectedPosition
+                ) { position, value ->  // Handle the SelectRowView item click listener
+                    viewModel.setPlaceHolderRowSelectedPosition(position)
+                }
+            }
+            //endregion
+
+            ExamplesSlot {
+                //region Example one
+                HmrcCard(modifier = Modifier.padding(bottom = HmrcTheme.dimensions.hmrcSpacing16)) {
+                    SelectRowView(
+                        selectRowViewItems = listOf(
+                            stringResource(R.string.select_row_view_first_row),
+                            stringResource(R.string.select_row_view_second_row),
+                            stringResource(R.string.select_row_view_error_row),
+                            stringResource(id = R.string.longer_text)
+                        ),
+                        errorText = exampleUiState.errorTextExample1,
+                        rowSelectedPosition = exampleUiState.rowSelectedPositionExample1
+                    ) { position, value -> // Handle the SelectRowView item click listener
+                        viewModel.setRowSelectedPositionExample1(position)
+                        if (position == THIRD_ROW_VIEW - 1) {
```

Comment:
This is the only usage of this const, so we could just set the const to the index of the item we want so we don't need to do additional calculations here.

```suggestion
           private const val ERROR_ITEM_INDEX = 2
                        if (position == ERROR_ITEM_INDEX) {
```